src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt

```diff
@@ -46,7 +48,7 @@ import org.sonarsource.sonarlint.plugin.api.module.file.ModuleFileEvent
  * The BulkFileListener is not tied to a specific project but global to the IDE instance
  */
 class VirtualFileSystemListener(
-    private val fileEventsNotifier: ModuleFileEventsNotifier = ModuleFileEventsNotifier()
+    private val fileEventsNotifier: ModuleFileEventsNotifier = ModuleFileEventsNotifier(),
```

Comment:
Is it from the sonar code style?

Reply:
Yes
![image](https://github.com/SonarSource/sonarlint-intellij/assets/64742703/a487211c-2cc9-4ff8-8291-e6253cbf5af8)

Reply:
I have the same but it does not add this comma 🤔